src/cgame/etj_speed_drawable.cpp

```diff
@@ -139,19 +321,30 @@ void ETJump::DisplaySpeed::render() const {
     frac = std::min(frac, 1.f);
 
     LerpColor(colorWhite, accelColor, color, frac);
-    // LerpColor adjusts alpha, make sure we still respect etj_speedAlpha
-    color[3] = etj_speedAlpha.value;
+  } else if (etj_speedColorUsesAccel.integer == 2) {
+    Vector4Copy(accelColor, color);
+  } else {
+    Vector4Copy(_color, color);
   }
 
-  CG_Text_Paint_Ext(x - w, y, size, size, color, status.c_str(), 0, 0, style,
+  CG_Text_Paint_Ext(speedX - speedW, speedY, speedSize, speedSize, color,
+                    status.c_str(), 0, 0, speedStyle,
                     &cgs.media.limboFont1);
+
+  if (etj_drawAccel.integer) {
+    CG_Text_Paint_Ext(accelX - accelW, accelY, accelSize, accelSize, color,
+                      accel_string.c_str(), 0, 0, accelStyle,
+                      &cgs.media.limboFont1);
+  }
```

Comment:
`canSkipDraw` will early out if `etj_drawSpeed2` is 0, so this might still not draw. Really this has no business being inside this speed drawable but I can take care of refactoring that later, this is fine for now as long as the drawing is fixed with `etj_drawSpeed2 0`.